File: apps/furniture-store-service-server/src/APIs/Product/Dtos/ProductCreateInput.cs
namespace FurnitureStoreService.APIs.Dtos;

public class ProductCreateInput
{
    public string? Id { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }

    public string? Name { get; set; }

    public string? Description { get; set; }

    public double? Price { get; set; }

    public Category? Category { get; set; }

    public List<OrderItem>? OrderItems { get; set; }
}

File: apps/furniture-store-service-admin/src/api/product/ProductCreateInput.ts
import { OrderItemCreateNestedManyWithoutProductsInput } from "./OrderItemCreateNestedManyWithoutProductsInput";

export type ProductCreateInput = {
  stock?: number | null;
  name?: string | null;
  description?: string | null;
  price?: number | null;
  orderItems?: OrderItemCreateNestedManyWithoutProductsInput;
};
